README.md

```diff
@@ -45,6 +46,9 @@ jobs:
   test:
     name: mabl Test
     runs-on: ubuntu-latest
+    environment: 
```

Comment:
Can you provide a link to the docs that say `environment` is _required_?

The [Workflows syntax indicates](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idenvironment) that this is optional. If there is no reference saying this is required, can we add a comment that this is _optional_, so users can keep their configs simple?